chef/automate
# Fix api docs to rename project-id to project_id

### :nut_and_bolt: Description: What code changed, and why?

Automate API docs for IAM has mentioned project-id in the examples where the correct parameter is project_id

### :chains: Related Resources

### :+1: Definition of Done
- The api docs when tested in local should have the correct parameter names

### :athletic_shoe: How to Build and Test the Change
- cd components/docs-chef-io
- make server
- Check localhost:

### :white_check_mark: Checklist

**All PRs** must tick these:


With occasional exceptions, all PRs **from Progress employees** must tick these:


All PRs **from Progress employees** should tick these if appropriate:


*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable
